input/fsh/Profiles clinical datapoints/BreastCancerPatient.fsh

```diff
@@ -28,6 +29,20 @@ Description: "The characteristics and demographic factors of an example patient.
 * extension[race].valueCodeableConcept  = RaceCS#2076-8
 * maritalStatus = NullFlavor#UNK
 
+Instance: BreastCancerPatient121
+InstanceOf: BreastCancerPatient 
+Title: "Example of Breast Cancer Patient"
+Description: "The characteristics and demographic factors of an example patient. The aim is to align with UScore or mCODE here in the future."
+* birthDate = "1933"
+* name.family = "Cornetet"
+* address.country = CountryCS#GB
+* gender = GenderCS#male
+* extension[educationLevel].valueCodeableConcept  = SCT#224297003
+* extension[ethnicity].valueCodeableConcept = EthnicityCS#2186-5
+* extension[race].valueCodeableConcept  = RaceCS#2054-5
+* maritalStatus = RelationshipStatusCS#D "Divorced"
+* deceasedDateTime = "1990-05-26"
```

Comment:
Can deceased[x] for the boolean "Vital status" also be added to the mapping? Does it work?